onnx/version_converter/BaseConverter.h

```diff
@@ -54,10 +54,10 @@ class BaseVersionConverter {
           if (adapter_ptr != target_map->second.end()) {
             return *(adapter_ptr->second);
           } else {
-            ONNX_ASSERTM(false, "No Adapter To Version %s for %s", initial.c_str(), op_name.c_str());
+            ONNX_ASSERTM(false, "No Adapter To Version %s for %s", target.c_str(), op_name.c_str());
           }
```

Comment:
Looks like the intended version number was `target` here. 

In the change below, I found it clearer to reword the message in that way.

Reply:
Good catch